Dockerfile

```diff
@@ -1,12 +1,10 @@
 FROM golang:1.12-alpine AS builder
 RUN apk --no-cache add git
-RUN go get github.com/constabulary/gb/...
 WORKDIR /build/
-COPY src/ /build/src/
-COPY vendor/ /build/vendor/
-RUN gb build all
+COPY . /build/
+RUN GO111MODULE=on go build
```

Comment:
`GO111MODULE` is not needed as this is not in `$GOPATH/src`, https://github.com/golang/go/wiki/Modules#how-to-use-modules